topics/variant-analysis/tutorials/sars-cov-2/tutorial.md

```diff
@@ -67,14 +93,14 @@ SRA can be reached either directly through it's website, or through the tool pan
 
 > ### {% icon comment %} Comment
 >
-> Initially the tool panel option exists only on the [usegalaxy.org server](https://usegalaxy.org/).  Support for the direct connection to SRA will be included in the 20.05 release of Galaxy
+> Initially the tool panel option for accessing SRA exists only on the [usegalaxy.org server](https://usegalaxy.org/).  Support for the direct connection to SRA will be included in the 20.05 release of Galaxy
```

Comment:
This comment seems outdated, can be removed?

Reply:
I'd leave this for now as we will be modifying this again once SRA fixes Run Selector